API/API/settings.py

```diff
@@ -124,3 +132,12 @@
 # https://docs.djangoproject.com/en/5.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+# TODO: remove the default key, as it may lead to security issues, also, maybe move this to a separate file, looks awful tbh
```

Comment:
Is this TODO comment just for you and you're going to resolve this during this PR in a separate commit? 
If so, then please mark this PR as a draft, because I don't know if I should review it. 
Otherwise, can you create a kanban item? Comments like this get lost oftentimes if they do not get saved somewhere.

Btw. this relates to the other TODO comment as well

Reply:
This being a TODO item is fine with me as long as it has a corresponding kanban item. 

Though sooner or later(and I think sooner) we'd want to have the ability to supply it as an env var (either the token or the path to it).

What I would do is check for the env variable and if it is there then use what's supplied and generate a key pair ad-hoc while maybe printing the public token to stdout otherwise

Reply:
I ended up generating the keys ad-hoc, just like you said.